### ISSUE TITLE
Error 500 on main domain activation for a WordPress Multisite

### ISSUE THREAD

**Issue by danicotillas** [Automattic/jetpack#5750]
Since the last update it´s impossible to work with Jetpack on the main domain of a Wordpress multisite instalation.
If i try to configure it, it bring me to a Error 500 page:
![captura de pantalla 2016-11-23 13 33 47](https://cloud.githubusercontent.com/assets/7635775/20572438/d282e6e4-b181-11e6-93b0-bc54b82a506e.png)

I hope there is a fix.
Thanks!

**Comment by jeherve** [comment]
I can't seem to reproduce on my end. Could you add the following to your site's `wp-config.php` file? 

```php
define( 'WP_DEBUG', true );

if ( WP_DEBUG ) {

        @error_reporting( E_ALL );
        @ini_set( 'log_errors', true );
        @ini_set( 'log_errors_max_len', '0' );

        define( 'WP_DEBUG_LOG', true );
        define( 'WP_DEBUG_DISPLAY', false );
        define( 'CONCATENATE_SCRIPTS', false );
        define( 'SAVEQUERIES', true );

}
```

Once you've done so, reproduce the error, and then check the `wp-content/debug.log` file for errors. You can paste the results here, and then replace `define('WP_DEBUG', true);` by `define('WP_DEBUG', false);` in the code above.

**Comment by danicotillas** [comment]
[23-Nov-2016 17:49:36 UTC] PHP Notice:  wpdb::escape is <strong>deprecated</strong> since version 3.6.0! Use wpdb::prepare() or esc_sql() instead. in /home1/comunil8/public_html/wp-includes/functions.php on line 3712
[23-Nov-2016 17:49:38 UTC] PHP Notice:  wpdb::escape is <strong>deprecated</strong> since version 3.6.0! Use wpdb::prepare() or esc_sql() instead. in /home1/comunil8/public_html/wp-includes/functions.php on line 3712
[23-Nov-2016 17:49:40 UTC] PHP Fatal error:  Call to undefined function PressForward\Core\API\register_rest_field() in /home1/comunil8/public_html/wp-content/plugins/pressforward/Core/API/APIWithMetaEndpoints.php on line 57
[23-Nov-2016 17:49:50 UTC] PHP Notice:  wpdb::escape is <strong>deprecated</strong> since version 3.6.0! Use wpdb::prepare() or esc_sql() instead. in /home1/comunil8/public_html/wp-includes/functions.php on line 3712
[23-Nov-2016 17:49:53 UTC] PHP Notice:  wpdb::escape is <strong>deprecated</strong> since version 3.6.0! Use wpdb::prepare() or esc_sql() instead. in /home1/comunil8/public_html/wp-includes/functions.php on line 3712
[23-Nov-2016 17:50:00 UTC] PHP Notice:  wpdb::escape is <strong>deprecated</strong> since version 3.6.0! Use wpdb::prepare() or esc_sql() instead. in /home1/comunil8/public_html/wp-includes/functions.php on line 3712
[23-Nov-2016 17:50:01 UTC] PHP Fatal error:  Call to undefined function PressForward\Core\API\register_rest_field() in /home1/comunil8/public_html/wp-content/plugins/pressforward/Core/API/APIWithMetaEndpoints.php on line 57
[23-Nov-2016 17:50:09 UTC] PHP Notice:  wpdb::escape is <strong>deprecated</strong> since version 3.6.0! Use wpdb::prepare() or esc_sql() instead. in /home1/comunil8/public_html/wp-includes/functions.php on line 3712
[23-Nov-2016 17:50:11 UTC] PHP Fatal error:  Call to undefined function PressForward\Core\API\register_rest_field() in /home1/comunil8/public_html/wp-content/plugins/pressforward/Core/API/APIWithMetaEndpoints.php on line 57

**Comment by jeherve** [comment]
`[23-Nov-2016 17:50:11 UTC] PHP Fatal error: Call to undefined function PressForward\Core\API\register_rest_field() in /home1/comunil8/public_html/wp-content/plugins/pressforward/Core/API/APIWithMetaEndpoints.php on line 57`

This seems to be the culprit. That's also a known issue with the Pressforward plugin (https://github.com/PressForward/pressforward/issues/859), and it will be fixed in Pressforward 4.2.

If you'd like, you can apply the following patch on your own site:
https://github.com/PressForward/pressforward/commit/85a18fb24030ae51bf0d2b43aeeff36a26825c0b
